lib/apis/v2/profile.go

```diff
@@ -48,7 +48,7 @@ type ProfileSpec struct {
 	// An option set of labels to apply to each endpoint (in addition to their own labels)
 	// referencing this profile.  If labels configured on the endpoint have keys matching those
 	// labels inherited from the profile, the endpoint label values take precedence.
-	LabelsToApply map[string]string `json:"labelsToApply,omitempty" protobuf:"bytes,11,rep,name=labels"`
+	LabelsToApply map[string]string `json:"labelsToApply,omitempty" protobuf:"bytes,11,rep,name=labels" validate:"omitempty,labelsToApply"`
```

Comment:
labelsToApply -> labels I think.  I don't think the validator needs to be something different to the standard labels validator does it?

Reply:
where did that protobufs incantation come from.  Not really sure it should be here.

Reply:
I suspect I added it by copying the labels field from the v1 metadata....

Reply:
yeah, I bet protobuf is wrong, will delete. when you say labelsToApply, you mean just the validator?

Reply:
btw, the regex is different:
`labelsToApplyValRegex = regexp.MustCompile("^[a-zA-Z0-9_.-]{0,128}$")`
`labelValueRegex       = regexp.MustCompile("^[a-zA-Z0-9]?([a-zA-Z0-9_.-]{0,61}[a-zA-Z0-9])?$")`

seems like gunjan went to some trouble to distinguish between the two types of labels.